app/controllers/user/registrations_controller.rb

```diff
@@ -0,0 +1,11 @@
+module User
+  class RegistrationsController < Devise::RegistrationsController
+    skip_before_action :verify_authenticity_token
+    respond_to :json
```

Comment:
add a space after this

Reply:
be consistent throughout your code, here you didn't add the space and in sessions_controller you got it right

Reply:
The example code uses the private visibility for the sessions_controller, before and after private the rubocop requires a space. I deleted the private visibility and the space.